<?xml version="1.0" encoding="UTF-8"?>
<?xml-stylesheet type="text/xsl" href="ivy-report.xsl"?>
<ivy-report version="1.0">
	<info
		organisation="org.scala-sbt.temp"
		module="temp-module-518b91374cbcb06003d475b7704c2316602396eb"
		revision="1.1.1"
		conf="runtime"
		confs="compile, runtime, test, provided, optional"
		date="20180427090429"/>
	<dependencies>
		<module organisation="org.scala-sbt" name="compiler-bridge_2.11">
			<revision name="1.1.1" error="public: unable to get resource for org/scala-sbt#compiler-bridge_2.11;1.1.1: res=https://repo1.maven.org/maven2/org/scala-sbt/compiler-bridge_2.11/1.1.1/compiler-bridge_2.11-1.1.1.pom: java.net.UnknownHostException: repo1.maven.org: Fallo temporal en la resoluciÃ³n del nombre" downloaded="false" searched="false" conf="default(compile)" position="-1">
				<caller organisation="org.scala-sbt.temp" name="temp-module-518b91374cbcb06003d475b7704c2316602396eb" conf="compile, runtime" rev="1.1.1" rev-constraint-default="1.1.1" rev-constraint-dynamic="1.1.1" callerrev="1.1.1"/>
				<artifacts>
				</artifacts>
			</revision>
		</module>
	</dependencies>
</ivy-report>
